src/scenes/Home/ParticipantHomeView/index.tsx

```diff
@@ -14,7 +14,7 @@ const ParticipantHomeView = () => {
 
   return (
     <PageWithModuleTimeline>
-      <Grid display={'flex'} flexDirection={'column'} flexGrow={1}>
+      <Grid display={'flex'} flexDirection={'column'} width={'100%'}>
```

Comment:
@karthikbtf unfortunately I had to put this back :( - the container wasn't shrinking correctly at lower resolutions.

Reply:
No worries, thanks for givint it a try earlier. This is totally fine if it solves the problem.